packages/core/src/getConfig.ts

```diff
@@ -0,0 +1,39 @@
+import { getBrowserJestConfig } from './browser/config/getBrowserJestConfig'
+import { getNodeJestConfig } from './node/config/getNodeJestConfig'
+
+// TODO: Add more types like eslint, babel, etc?
+// TODO: Move somewhere else
+enum ConfigType {
+  Jest = 'jest',
+}
+
+// TODO: Add more build targets
+// TODO: Move somewhere else
+enum BuildTarget {
+  Browser = 'browser',
+  Node = 'node',
+}
```

Comment:
We have something like this over here, which you can use:
https://github.com/redwoodjs/redwood/blob/main/packages/internal/src/config.ts#L8-L13